### ISSUE TITLE
Can't set SAVE_FAVICON=false: Error while loading configuration value: USE_CURL — KeyError: 'FETCH_TITLE'

### ISSUE THREAD

**Issue by jwilk** [pirate/ArchiveBox#400]
```console
$ archivebox config --set SAVE_FAVICON=false
[i] [2020-07-30 08:56:28] ArchiveBox v0.4.9: archivebox config --set SAVE_FAVICON=false
    > /home/jwilk/archivebox


[X] Error while loading configuration value: USE_CURL
    KeyError: 'FETCH_TITLE'
```

**Comment by pirate** [comment]
Can you post your `ArchiveBox.conf` file here

**Comment by pirate** [comment]
Whoops sorry nevermind. Fixed it: 5707ffe.

You can get the fix early before the next release using:
```bash
python -m pip install git+https://github.com/pirate/ArchiveBox.git@master
```
